### PR TITLE
add option flags -O0 to ./julia

### PR DESCRIPTION
**Pull request by vtjnash**
this ensures all local variables live in stack slots, which makes debugging more pleasant and may run OrcJIT tests much faster

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -95,8 +95,8 @@ script:
         /tmp/julia/bin/julia-debug -J local.ji -e 'true' && rm local.ji
     - /tmp/julia/bin/julia -e 'versioninfo()'
     - export JULIA_CPU_CORES=2 && export JULIA_TEST_MAXRSS_MB=600 && cd /tmp/julia/share/julia/test &&
-        /tmp/julia/bin/julia --check-bounds=yes runtests.jl $TESTSTORUN &&
-        /tmp/julia/bin/julia --check-bounds=yes runtests.jl libgit2-online pkg
+        /tmp/julia/bin/julia -O0 --check-bounds=yes runtests.jl $TESTSTORUN &&
```
Comment (yuyichao) [inline .travis.yml]: I might not be a good idea to run the CI with non-default setting though...

Comment (carnaval) [inline .travis.yml]: yep I think it's just a temporary way of seeing if it breaks anything

**Comment by JeffBezanson** [comment]
Nice!

Just to see if we're happy with the names, do we want `-O0` for this mode, and `-O1` for, say, only running mem2reg?

File: .travis.yml (hunk 1 above)
Comment (yuyichao) [inline .travis.yml]: OK. Sure.

Comment (tkelman) [inline .travis.yml]: shouldn't this be reverted?

Comment (tkelman) [inline .travis.yml]: Bump. This doesn't seem to make Travis noticeably faster, it would be better to test the more commonly used case.

**Comment by ViralBShah** [comment]
I like the suggested names for the passes.

**Pull request merged**